src/Webapi/Webapi__Navigator.res

```diff
@@ -0,0 +1,45 @@
+type t
```

Comment:
I'm working on some test changes in this area and I just realised, now that this type isn't shared with `WorkerNavigator` would you be open to swapping them both to a record type?

The `@send` apis will still need to be external, but I believe every `@get` external can just be a record field.

Reply:
Sure, that's fine with me. Were you asking because you wanted to do it in your branch or asking if I'd make a PR separately? Either is fine with me

Reply:
I’m happy to do it, I just wanted to make sure it wouldn’t cause problems for you :)

Reply:
Nope, haven't had time to migrate to this from bs-webapi yet, but gonna have to do that soon